evergreen/install-dependencies.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -o errexit  # Exit the script with error if any of the commands fail
 
-DOTNET_SDK_PATH=./.dotnet
+DOTNET_SDK_PATH="${DOTNET_SDK_PATH:-./.dotnet}"
```

Comment:
Need this to control the dotnet installation path from outside. Used when the script is being run as part of Driver's CI.